src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -148,15 +149,15 @@ struct PlantHydraulicsParameters{
     conductivity_model::CP
     "Water retention model and parameters"
     retention_model::RP
-    "Root distribution function P(z)"
-    root_distribution::F
+    "Rooting depth parameter (m) - used to calculate root_distribution"
```

Comment:
Maybe:
"Rooting depth parameter (m) - a characteristic depth below which `1/e` of the root mass lies."